### PR TITLE
Add Windows build (try win iii?)

### PR DESCRIPTION
**Pull request by brynpickering**
Trying out whether @wolfv's implementation in #3 could now work with updates to the Fortran compiler. I don't hold out too much hope given the issues with file extensions [raised in that PR](https://github.com/conda-forge/coin-or-utils-feedstock/pull/3#issuecomment-752748604), but creating this PR to trigger the CI and see what's going on.

Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by brynpickering** [comment]
@conda-forge-admin, please rerender

**Comment by jpfeuffer** [comment]
@conda-forge-admin, please rerender

**Comment by jpfeuffer** [comment]
There is something very wrong. I don't think this is how you build on Windows by just executing the build.sh in build.bat.

Especially if it needs to load vcvarsall.bat into the environment

File: recipe/bld.bat [hunk 1]
```diff
@@ -0,0 +1 @@
+bash "%RECIPE_DIR%\build.sh"
```
Comment (jpfeuffer) [inline recipe/bld.bat:1]:
```suggestion
set "OPENBLAS_NUM_THREADS=1"

if "%LIBRARY_PREFIX%"=="" (
    set "USE_PREFIX=%PREFIX%"
    set "WITH_BLAS_LIB=-L%PREFIX%\lib -lblas"
    set "WITH_LAPACK_LIB=-L%PREFIX%\lib -llapack"
    set "CFLAGS=%CFLAGS% -O3"
    set "CXXFLAGS=%CXXFLAGS% -O3"
    set "CXXFLAGS=%CXXFLAGS:-std=c++17=-std=c++11%"
) else (
    set "USE_PREFIX=%LIBRARY_PREFIX%"
    set "WITH_BLAS_LIB=%LIBRARY_PREFIX%\lib\blas.lib"
    set "WITH_LAPACK_LIB=%LIBRARY_PREFIX%\lib\lapack.lib"
)

set "WIN_FLAGS=F77=flang --build=x86_64-w64-mingw32 --enable-msvc"

.\configure ^
    --prefix="%USE_PREFIX%" ^
    --exec-prefix="%USE_PREFIX%" ^
    --with-blas="%WITH_BLAS_LIB%" ^
    --with-lapack="%WITH_LAPACK_LIB%" ^
    %WIN_FLAGS%

make -j "%CPU_COUNT%"
make install
```

File: recipe/build.sh [hunk 2]
```diff
@@ -1,13 +1,10 @@
 #!/usr/bin/env bash
-# Get an updated config.sub and config.guess
-cp $BUILD_PREFIX/share/gnuconfig/config.* ./CoinUtils
-cp $BUILD_PREFIX/share/gnuconfig/config.* .
 set -e
 
+# for win
+mkdir -p /tmp
```
Comment (jpfeuffer) [inline recipe/build.sh:5]:
```suggestion
/bin/mkdir -p /tmp
```

**Pull request closed without merging**